openstack/dds/v3/instances/urls.go

```diff
@@ -21,3 +21,11 @@ func modifyURL(c *golangsdk.ServiceClient, serverID, action string) string {
 func portModifiedURL(c *golangsdk.ServiceClient, instanceId string) string {
 	return c.ServiceURL("instances", instanceId, "modify-port")
 }
+
+func secondsLevelMonitoringModifiedURL(c *golangsdk.ServiceClient, instanceId string) string {
```

Comment:
you can refer to the API Explorer
```suggestion
func secondLevelMonitoringURL(c *golangsdk.ServiceClient, instanceId string) string {
```

Reply:
done